srcpkgs/uhd/template

```diff
@@ -0,0 +1,39 @@
+# Template file for 'uhd'
+pkgname=uhd
+version=3.13.0.2
+revision=1
+build_style=cmake
+hostmakedepends="doxygen"
+makedepends="python-Mako libusb-devel python-numpy python-devel boost-devel"
+short_desc="Ettus USRP Hardware driver library"
+maintainer="Remi Pommarel <repk@triplefau.lt>"
+license="GPL-3.0-or-later"
+homepage="https://files.ettus.com/"
+distfiles="https://github.com/EttusResearch/uhd/archive/v${version}.tar.gz"
+checksum=e18d0524cbf571be4847fd7f971dc30c37efd9e7a333761b74e1266a07cbd35b
+wrksrc=${pkgname}-${version}
+build_wrksrc=host
+
+post_install() {
+	rm -rf ${DESTDIR}/usr/lib/uhd/tests
+}
+
+uhd-devel_package() {
+	depends="${makedepends} ${sourcepkg}-${version}_${revision}"
```

Comment:
only dependency arrcoding to pkgconfig file is gnuradio-devel

Reply:
Are you sure ? I mean this package is fine without gnuradio-devel. It's gnuradio that needs it not the otherway around.

Reply:
Yes, xpcdeps seems to be borked in that regard.